.storybook/story-wrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+
+import "./global.less";
+
+/**
+ * Wrapper for Storybook stories that
+ * 1. Renders the story inside of a .framework-perseus container
+ * 2. Includes the global styles from prod
+ */
+function StoryWrapper(props) {
+    // Most of our components have an expectation to be
+    // rendered inside of a .framework-perseus container.
+    // We want to make sure we can include it here, since it
+    // can also affect the styling.
+    return (
+        <div className="framework-perseus box-sizing-border-box-reset">
+            {props.children}
+        </div>
```

Comment:
Do we need a separate file for this? Could we not just wrap the `preview.tsx` content in this `<div>` directly?

Reply:
Ah yeah. Now that I removed the import, I'll change it back to how it was before.